rllib/algorithms/algorithm_config.py

```diff
@@ -319,26 +319,37 @@ def __init__(self, algo_class=None):
         # If not specified, we will try to auto-detect this.
         self._is_atari = None
 
+        # TODO (sven): Rename this method into `AlgorithmConfig.sampling()`
```

Comment:
Now that we are aiming for a the EnvRunner API as the default, we should rename/clarify some of these config settings and methods.